src/apps/answers/filters.py

```diff
@@ -64,3 +69,19 @@ class AppletMultiinformantAssessmentParams(InternalModel):
     target_subject_id: uuid.UUID | None
     source_subject_id: uuid.UUID | None
     activity_or_flow_id: uuid.UUID | None
+
+    @root_validator
+    def validate_assignments(cls, values):
+        target = values.get("target_subject_id")
+        source = values.get("source_subject_id")
+        # Target, source, and activity/flow must all be present
+        if not target and not source:
+            raise MultiinformantAssessmentMissingTargetAndSource()
+
+        if not target:
+            raise MultiinformantAssessmentInvalidTargetSubject()
+
+        if not source:
+            raise MultiinformantAssessmentInvalidSourceSubject()
+
+        return values
```

Comment:
Agreed with @rcmerlo – we should just use normal Pydantic validation the way the BE does when validating other requests. `422 Unprocessable Entity` is a totally fine error response for omitting any of these values.

```suggestion
    target_subject_id: uuid.UUID
    source_subject_id: uuid.UUID
    activity_or_flow_id: uuid.UUID
```

Reply:
Resolve on last commit, Thanks Rodrigo